I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 374, in script
    call screen emerald_button
  File "renpy/common/000statements.rpy", line 671, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "renpy/common/00action_control.rpy", line 172, in __call__
    renpy.show_screen(self.screen, *self.args, **self.kwargs)
Exception: Screen redButton is not known.


-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 374, in script
    call screen emerald_button
  File "/Applications/renpy-8.3.3-sdk/renpy/ast.py", line 1974, in execute
    self.call("execute")
  File "/Applications/renpy-8.3.3-sdk/renpy/ast.py", line 1953, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/Applications/renpy-8.3.3-sdk/renpy/statements.py", line 374, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 671, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/Applications/renpy-8.3.3-sdk/renpy/exports/statementexports.py", line 345, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/Applications/renpy-8.3.3-sdk/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Applications/renpy-8.3.3-sdk/renpy/display/core.py", line 3286, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/screen.py", line 793, in event
    rv = self.child.event(ev, x, y, st)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/behavior.py", line 1182, in event
    return handle_click(self.clicked)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/behavior.py", line 1103, in handle_click
    rv = run(action)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/behavior.py", line 394, in run
    new_rv = run(i, *args, **kwargs)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_control.rpy", line 172, in __call__
    renpy.show_screen(self.screen, *self.args, **self.kwargs)
  File "/Applications/renpy-8.3.3-sdk/renpy/display/screen.py", line 1327, in show_screen
    raise Exception("Screen %s is not known.\n" % (name[0],))
Exception: Screen redButton is not known.


macOS-14.4-arm64-arm-64bit arm64
Ren'Py 8.3.3.24111502
final_project_wizard_of_oz 1.0
Thu Dec 12 18:15:16 2024
